enterprise/api/v1/views.py

```diff
@@ -27,6 +27,14 @@ class EnterpriseReadOnlyModelViewSet(viewsets.ReadOnlyModelViewSet):
     throttle_classes = (ServiceUserThrottle,)
 
 
+class EnterpriseCreateListRetrieveUpdateModelViewSet(
```

Comment:
From View name it is not clear which models this view set belongs to, can you update the name to something else ?